mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -692,6 +697,20 @@ LogicalResult getTuningProblemStr(rock::RockGemmWrapperInterface gemmIF,
     oLayout[oLayoutMap["1o"]] = '1';
     oLayout[oLayoutMap["go"]] = 'G';
 
+    if (size > 5) {
```

Comment:
How about this:

Go to the code above, get rid of the `"0"`, `"0i"`, etc. cases.

Then,
```c++
for(unsigned img = 0, nImg = size - 3; img < nImg, ++nImg) {
  fLayout[fLayoutMap[llvm::formatv("{}", key)]] = '0' + i;
  iLayout[iLayoutMap[llvm::formatv("{}i", key)]] = '0' + i;
  oLayout[oLayoutMap[llvm::formatv("{}o"}, key)]] = '0' + i;
}
```

Reply:
The loop as shown doesn't make much sense, but I think I did what you were thinking of.